doc/changes/latest.inc

```diff
@@ -60,6 +60,7 @@ Bugs
 - Fix bug where annotation FIF files lacked an end block tag (:gh:`11800` by `Eric Larson`_)
 - Fix display of :class:`~mne.Annotations` in `mne.preprocessing.ICA.plot_sources` when the ``raw`` has ``raw.first_samp != 0`` and doesn't have a measurement date (:gh:`11766` by `Mathieu Scheltienne`_)
 - Fix bug in read_raw_eyelink, where STATUS information of samples was always assumed to be in the file. Performance and memory improvements were also made. (:gh:`11823` by `Scott Huberty`_)
+- Fix bug in convert_flash_mris, when flash30 and no flash5_reg.mgz file regenrated, add missing_ok=True as it is done in line 2074. (:gh:`11842` by `Hamza Abdelhedi`_)
```

Comment:
You'll also need to add yourself in `names.inc` (https://github.com/mne-tools/mne-python/blob/main/doc/changes/names.inc) to get the link working.

Reply:
@mscheltienne  done, is there any other thing I need to do? or a list of steps ?

Reply:
If you're a new contributor you should use the newcontrib tag an move the entry to the top of the list

Reply:
@agramfort  done